src/lib.rs

```diff
@@ -189,6 +203,15 @@ impl Window {
     }
 }
 
+impl Notifier for Window {
+    type Event = Widget;
```

Comment:
After those changes, we'll want to change `Event` to `Action` because the window will notify the program when an action occurs.